File: app/src/main/java/com/mona/moviesapp/popular_people/FullImageScreen/model/FullImageModel.java
package com.mona.moviesapp.popular_people.FullImageScreen.model;

import com.mona.moviesapp.popular_people.FullImageScreen.controller.FullImageController;

public class FullImageModel {

    FullImageController fullImageController;
    FullImageNetwork fullImageNetwork = new FullImageNetwork(this, fullImageController);

    public FullImageModel(FullImageController fullImageController) {
        this.fullImageController = fullImageController;
        this.fullImageNetwork = fullImageNetwork;
    }

    public void setPicturePath(String picturePath) {
        fullImageNetwork.execute(picturePath);
    }
}

File: app/src/main/java/com/mona/moviesapp/popular_people/FullImageScreen/model/FullImageModel.kt
package com.mona.moviesapp.popular_people.FullImageScreen.model

import com.mona.moviesapp.popular_people.FullImageScreen.controller.FullImageController

class FullImageModel(internal var fullImageController: FullImageController) {
    internal var fullImageNetwork = FullImageNetwork(this, fullImageController)

    init {
        this.fullImageNetwork = fullImageNetwork
    }

    fun setPicturePath(picturePath: String) {
        fullImageNetwork.execute(picturePath)
    }
}
